meeting-search.php

```diff
@@ -65,11 +65,12 @@
         if ($results_counter == $results_count) break;
     }
 
-    echo "<Pause length=\"2\"/>";
+    
 
     // Do not handle for the SMS gateway
     if (!isset($_REQUEST["SmsSid"]) && count($filtered_list) > 0) {
-        if (count($sms_messages) > 0) { ?>
+        if (count($sms_messages) > 0) { 
```

Comment:
can you just move to have it below the first `if`?

Reply:
only need one instance of this.

Reply:
yes that makes much more sense